### ISSUE TITLE
weights

### ISSUE THREAD

**Issue by esdgdhln** [kentsommer/tensorflow-posenet#4]
why we need a start weights to train?

**Comment by kentsommer** [comment]
Hi @esdgdhln,

The starting weights are used to fine-tune the posenet model. You, of course, can train from scratch, however, the performance will be lower (as discussed in the paper).

**Comment by esdgdhln** [comment]
Yes,besides,I cannot access the pagelink to /pvt/GoogLeNet/Places/,can you seed me a cope？Thank you!

**Comment by kentsommer** [comment]
@esdgdhln 

They changed the url, I have already updated it in the readme, it is now: http://3dvision.princeton.edu/pvt/GoogLeNet/Places/
